hpat/io/_csv.cpp

```diff
@@ -309,14 +309,14 @@ static PyObject* csv_chunk_reader(std::istream * f, size_t fsz, bool is_parallel
         std::vector<size_t> line_offset = count_lines(f, hpat_dist_get_node_portion(fsz, nranks, rank));
         size_t no_lines = line_offset.size();
         // get total number of lines using allreduce
-        size_t tot_no_lines(0);
+        int64_t tot_no_lines = 0;
 
         hpat_dist_reduce(reinterpret_cast<char *>(&no_lines), reinterpret_cast<char *>(&tot_no_lines), HPAT_ReduceOps::SUM, HPAT_CTypes::UINT64);
 
         // Now we need to communicate the distribution as we really want it
         // First determine which is our first line (which is the sum of previous lines)
-        size_t byte_first_line = hpat_dist_exscan_i8(no_lines);
-        size_t byte_last_line = byte_first_line + no_lines;
+        int64_t byte_first_line = hpat_dist_exscan_i8(no_lines);
+        int64_t byte_last_line = byte_first_line + no_lines;
```

Comment:
why do change to signed?

Reply:
Because later it participated in comparison with signed input argument. In this case tne compiler show the warning.